AK/FixedArray.h

```diff
@@ -160,6 +160,13 @@ class FixedArray {
         ::swap(m_elements, other.m_elements);
     }
 
+    void fill_with(T value)
+    {
+        // FIXME: Let's hope the compiler optimizes this, otherwise we could do some trickery for trivial classes.
```

Comment:
The compiler should definitely be smart enough to optimize this. I'd recommend looking at the assembly before inserting FIXMEs. Ideally FIXMEs should be definite directives to fix an already-known issue.